src/dotnet-ildasm/Disassembler.cs

```diff
@@ -136,7 +138,8 @@ private void WriteTypeSignature(TypeDefinition type)
             //TODO: Signature to use IL types #2
             _outputWriter.Write($" {type.FullName}");
             //TODO: External Types should always be preceded by their assembly names #6
-            _outputWriter.Write($" extends {type.BaseType.FullName}");
+            if(!type.IsInterface)
```

Comment:
if the TypeDefinition object is a interface type, the BaseTye will be null and it will throw a System.NullReferenceException triying to access to the FullName

![image](https://cloud.githubusercontent.com/assets/10756713/26479012/31176bde-420b-11e7-94f1-652b3e9f6f43.png)